app/logic/Auth/LoginAutoFill.ts

```diff
@@ -1,6 +1,7 @@
+import type { Account } from "../Abstract/Account";
 
-export function owaAutoFillLoginPage(username = "", password = "") {
-  return `(${autoFillPageScript})(${JSON.stringify(username)}, ${JSON.stringify(password)})`;
+export function autoFillLoginPage(account: Account) {
+  return `(${autoFillPageScript})(${JSON.stringify(account.username || "")}, ${JSON.stringify(account.password || "")})`;
```

Comment:
```suggestion
  return `(${autoFillPageScript})(${JSON.stringify(account.username ?? "")}, ${JSON.stringify(account.password ?? "")})`;
```

Please use `??` unless you want to cover the var being `""`.